### PR TITLE
iOS. Fixed bug with selection handle

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Previously SelectionHandle looks like this on BasicTextField:

image

It was because of Popup padding

Not, it was fixed:

image

### CODE REVIEW

File: compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt [hunk 1]
```diff
@@ -25,5 +25,6 @@ val IosBugs = Screen.Selection(
     KeyboardPasswordType,
     UIKitRenderSync,
     DispatchersMainDelayCheck,
-    StartRecompositionCheck
+    StartRecompositionCheck,
+    SelectionHandlePadding,
```
Comment (MatkovIvan) [inline compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt]: Is it iOS only?

File: compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/selection/SelectionHandles.uikit.kt [hunk 2]
```diff
@@ -52,6 +54,7 @@ private val RADIUS = 6.dp
  */
 private val THICKNESS = 2.dp
 
+@OptIn(ExperimentalComposeUiApi::class)
```
Comment (MatkovIvan) [inline compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/selection/SelectionHandles.uikit.kt:57]: BTW It's cross-module experimental, but that's not first usage of it. So I guess it's OK here.

File: compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt (hunk 1 above)
Comment (MatkovIvan) [reply compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt]: Please move it to common, there is no specific uikit API usage here.

Comment (dima-avdeev-jb) [reply compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt]: Code sample not iOS only. So, moved to commonMain sourceSet in TextFields section

File: compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/selection/SelectionHandles.uikit.kt (hunk 2 above)
Comment (dima-avdeev-jb) [reply compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/selection/SelectionHandles.uikit.kt:57]: Thanks for notice about it!

File: compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt (hunk 1 above)
Comment (dima-avdeev-jb) [reply compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt]: Done

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**